ddl/ddl_worker.go

```diff
@@ -843,10 +843,12 @@ func updateSchemaVersion(t *meta.Meta, job *model.Job) (int64, error) {
 		}
 		diff.TableID = tbInfo.ID
 	case model.ActionRenameTable:
-		err = job.DecodeArgs(&diff.OldSchemaID)
+		oldSchemaIDs := []int64{}
+		err = job.DecodeArgs(&oldSchemaIDs)
 		if err != nil {
 			return 0, errors.Trace(err)
 		}
+		diff.OldSchemaID = oldSchemaIDs[0]
```

Comment:
We can use `AffectedOption` to handle it. e.g. `rename table db1.t1 to db3.t3, db2.t2 to db4.t4`.
```
SchemaDiff{
...
        SchemaID -> db3.ID
        TableID -> tbl3.ID
        OldTableID -> tbl1.ID
        OldSchemaID -> db1.ID
}

SchemaDiff.AffectedOpts[0] = &AffectedOption{oldSchemaIDs[0], newSchemaIDs[0], tableIDs[0]}
```

And we can add some tests like `TestAlterTableRenameTable` in ddl/db_test.go to check.